deployment/deployment.json

```diff
@@ -3855,6 +3855,58 @@
       }
     }
   },
+  "pancakeswap-v3-swaps": {
```

Comment:
I am sorry if I confused you, I already deployed with the word `swap` and not `swaps`...

Reply:
my bad, corrected.